### PR TITLE
Feature edit user

### PR DESCRIPTION
**Pull request by PierrePaul**
https://trello.com/c/2bInxmlN/105-implement-a-password-reset-screen
https://trello.com/c/5kncrAdX/202-allow-users-to-update-their-name-through-their-profile
https://trello.com/c/owGfs1Hk/201-allow-users-to-update-their-mobile-number-through-their-profile

https://www.figma.com/file/znPqLTQQDcUkaNjdDcShiz/Exposure-Notification---Health-Portal-(mock-up)?node-id=0%3A1

This PR let the user change all his information, except the province (which is no longer shown).
Managers can also change someone else info, if they are in the same province.

### CODE REVIEW

**Comment by pcraig3** [comment]
One more update!

In the Figma mocks, we have an extra "Edit" link beside delete.

image (2)

Let's add that instead of making the account name a link.

image (1)

File: profiles/forms.py [hunk 1]
```diff
@@ -73,6 +73,101 @@ class Healthcare2FAForm(HealthcareBaseForm):
...
+
+class HealthcarePasswordEditForm(HealthcareBaseEditForm):
+    title = _("Change your password")
+    error_messages = {
+        "password_mismatch": _("The two password fields didn’t match."),
+    }
+    password1 = forms.CharField(
+        label=_("Password"),
+        strip=False,
+        widget=forms.PasswordInput(attrs={"autocomplete": "new-password"}),
```
[the remark this answers is not in the document]
Comment (PierrePaul) [reply profiles/forms.py:129]: I feel like this is a copy paste and my main branch was not updated at the time. Removing.

File: profiles/templates/profiles/healthcareuser_detail.html [hunk 2]
```diff
@@ -21,23 +21,45 @@ <h1>{% trans "User account" %}</h1>
       <caption class="visually-hidden">{% trans "User account" %}</caption>
       <tbody>
         <tr>
-          <th scope="row">{% trans "Your name" %}</td>
+          <th scope="row">{% trans "Your name" %}</th>
```
Comment (pcraig3) [inline profiles/templates/profiles/healthcareuser_detail.html:24]: good catch ⚾

File: profiles/templates/profiles/healthcareuser_list.html [hunk 3]
```diff
@@ -23,13 +23,19 @@ <h1>{% trans "Manage team" %}</h1>
...
           <td>
             <strong class="tag tag--{{ healthcareuser.is_admin|yesno:'green,blue' }}">
               {{ healthcareuser.is_admin|yesno:_("Admin,Staff") }}
             </strong>
           </td>
           <td>
+          {% if not healthcareuser.is_superuser and healthcareuser.id != user.id %}
+              <a href="{% url 'user_profile' healthcareuser.id %}">
+                  <span aria-hidden="true">{% trans "Edit" %}</span>
+                  <span class="visually-hidden">{% blocktrans with name=healthcareuser.name %}Edit {{ name }} account{% endblocktrans %}</span>
```
Comment (pcraig3) [inline profiles/templates/profiles/healthcareuser_list.html:36]: Can we make this like the one below?

> Edit account for {{ email }}

File: locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -689,12 +691,21 @@ msgstr "Droit d’accès"
 msgid "Admin,Staff"
 msgstr "Administrateur,Employé"
 
+#: profiles/templates/profiles/healthcareuser_list.html:35
+msgid "Edit"
+msgstr ""
```
Comment (pcraig3) [inline locale/fr/LC_MESSAGES/django.po:696]: I think it's "Modifier"

File: profiles/templates/profiles/healthcareuser_list.html [hunk 5]
```diff
@@ -23,13 +23,19 @@ <h1>{% trans "Manage team" %}</h1>
...
+          <td>{{ healthcareuser.email }}</td>
           <td>
             <strong class="tag tag--{{ healthcareuser.is_admin|yesno:'green,blue' }}">
               {{ healthcareuser.is_admin|yesno:_("Admin,Staff") }}
             </strong>
           </td>
           <td>
+          {% if not healthcareuser.is_superuser and healthcareuser.id != user.id %}
+              <a href="{% url 'user_profile' healthcareuser.id %}">
+                  <span aria-hidden="true">{% trans "Edit" %}</span>
```
Comment (pcraig3) [inline profiles/templates/profiles/healthcareuser_list.html:35]: There's an extra space you see in the browser (`Edit `) you can see the underline is too long, so let's change the markup to try and remove it.

**Pull request merged**